cms/articles/models.py

```diff
@@ -231,6 +249,16 @@ def get_admin_display_title(self) -> str:
         """Changes the admin display title to include the parent title."""
         return f"{self.get_parent().title}: {self.draft_title or self.title}"
 
+    def get_headline_figure(self, figure_id: str) -> dict[str, str]:
```

Comment:
needs unit test

Reply:
Test added in [e564a03](https://github.com/ONSdigital/dis-wagtail/pull/152/commits/e564a03ec4ca4e78b3f6e330908e60572392685e)